content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+
+To help demystify the nature of CVEs within Chainguard Images, we've created a self-service [Security Advisories page](https://images.chainguard.dev/security) that lists every security advisory published for Chainguard Images. Having this information available allows you to view whether Chainguard is aware of a specific vulnerability reported to exist within a Chainguard Image and whether we've mitigated or are planning to mitigate the CVE.
+
+This guide outlines how you can use Chainguard's Security Advisories to learn more about the status of a CVE within a given package. It will walk through a practical example of discovering a vulnerability in a Chainguard Image, searching for Security Advisories associated with this vulnerability, and then comparing the original Image with a later version.
+
+
+## Prerequisites
+
+You don't need any special access or software to explore Chainguard's Security Advisories. However, this guide includes a few examples that use specific software tools in order to outline a practical example of how one might navigate and use the Security Advisories.
+
+To follow along with these examples, you'll need the following tools installed.
+
+* [Docker Scout](https://docs.docker.com/scout/) — A component of the Docker container platform, Docker Scout allows you to scan container images to identify potential vulnerabilities. To use Docker Scout, you will need to [install Docker](https://docs.docker.com/get-docker/) on your chosen platform.
+* [`chainctl`](/chainguard/chainctl/) — Chainguard's command-line interface tool. To install `chainctl`, follow our [installation guide](/chainguard/administration/how-to-install-chainctl/). 
+* [`jq`](https://jqlang.github.io/jq/) — `jq` is a command-line JSON processor that allows you to filter and manipulate streaming JSON data. Although it isn’t strictly necessary for the purposes of this guide, this tutorial includes commands that use `jq` to filter command output that would otherwise be difficult to read. You can install `jq` by following the instructions on [the project’s Download jq page](https://jqlang.github.io/jq/download/).
+
+
+## So you've encountered a CVE in a Chainguard Image
+
+Say you use a vulnerability scanner like Grype or Docker Scout to inspect a certain Chainguard Image. This example uses Docker Scout to scan a Chainguard Production Image, specifically `go:1.21.2`.
+
+```sh
+docker scout cves cgr.dev/chainguard-private/go:1.21.2
+```
+
+Because this is the digest for an older version of Chainguard's Go Image, this command's output will show a number of vulnerabilities that have been found to exist within this specific version of the Image. 
```

Comment:
"this is the digest" -- you didn't specify a digest. I think it would probably best if you did; the tag 1.21.2 will be withdrawn at some point, but the digest would remain available.

Reply:
I'd also add something like "all of these vulnerabilities have been addressed in newer versions of the image"

Reply:
good points all around!